templates/home/index.html.twig

```diff
@@ -1,8 +1,56 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}Home{% endblock %}
+{% block stylesheets %}
+    {{ parent() }}
+    {{ encore_entry_link_tags('home') }}
+{% endblock %}
+
+{% block title %}{{ website }} - Home{% endblock %}
 
 {% block body %}
-    <h1>Welcome on your project 3</h1>
-    <img src="https://media.giphy.com/media/hS58Kb2OOCMPS/giphy.gif" alt="Dewey dancing">
+    <section id='home'>
+        <div class='container_home-menu' id="menu">
+            <img id='home-logo' src="{{ asset('build/images/logo.svg')}}" alt="logo">
```

Comment:
Your static images should probably not be in the build directory. May be in public/images ?

Reply:
OK done